src/main/kotlin/com/example/delivery/config/DatabaseChangelog.kt

```diff
@@ -0,0 +1,52 @@
+package com.example.delivery.config
+
+import com.mongodb.client.MongoDatabase
+import io.mongock.api.annotations.ChangeUnit
+import io.mongock.api.annotations.Execution
+import io.mongock.api.annotations.RollbackExecution
+import org.bson.Document
+import org.slf4j.LoggerFactory
+
+@ChangeUnit(id = "OrderIndexCreationChangelog", order = "001", author = "torbin.v@ajax.systems")
+class DatabaseChangelog {
+
+    @Execution
+    fun createIndexOnUserCollection(db: MongoDatabase) {
+        val orderCollection = db.getCollection("order")
+
+        val indexDocument = Document("userId", 1)
+
+        orderCollection.createIndex(indexDocument)
+    }
+
+    @RollbackExecution
+    fun rollback(db: MongoDatabase) {
+        log.atInfo()
```

Comment:
move id = OrderIndexCreationChangelog to the constant and use it both in `@ChangeUnit` and in LOG arg. Log looks horrible